pyproject.toml

```diff
@@ -12,6 +12,7 @@ exclude_rules = [
     "references.keywords",
     "references.consistent",
     "references.special_chars",
+    "layout.cte_newline",
```

Comment:
I prefer

```sql
WITH a AS (
...
), b AS (
...
)
```

to

```sql
WITH a AS (
...
), 

b AS (
...
)
```

Reply:
I do like the newlines between CTEs. seems to really help readability since each CTE is a logical chunk that could be pretty long with no newlines in it